This has some an issue with tsc runs:

```
packages/a11y/src/index.js:4:22 - error TS2307: Cannot find module '@wordpress/dom-ready'.

4 import domReady from '@wordpress/dom-ready';
                       ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

We'll need to figure out how to make tsc aware of other packages, likely along the lines of https://github.com/WordPress/gutenberg/issues/18838#issuecomment-560014005

--- a/packages/a11y/src/addContainer.js
+++ b/packages/a11y/src/addContainer.js
@@ -29,9 +29,11 @@ const addContainer = function( ariaLive ) {
 	container.setAttribute( 'aria-relevant', 'additions text' );
 	container.setAttribute( 'aria-atomic', 'true' );
 
-	// Disable reason: the querySelector + optional chain may make the expression `null`.
-	// eslint-disable-next-line no-unused-expressions
-	document.querySelector( 'body' )?.appendChild( container );
+	const body = document.querySelector( 'body' );
+	if ( body ) {
+		body.appendChild( container );
+	}
+
 	return container;
 };
 

--- a/packages/a11y/src/addContainer.js
+++ b/packages/a11y/src/addContainer.js
@@ -3,7 +3,7 @@
  *
  * @param {string} ariaLive Optional. Value for the 'aria-live' attribute, default 'polite'.
  *
- * @return {HTMLDivElement} The ARIA live region jQuery object.
+ * @return {HTMLDivElement} The ARIA live region HTML element.
  */
 const addContainer = function( ariaLive ) {
 	ariaLive = ariaLive || 'polite';

--- a/packages/a11y/src/addContainer.js
+++ b/packages/a11y/src/addContainer.js
@@ -3,7 +3,7 @@
  *
  * @param {string} ariaLive Optional. Value for the 'aria-live' attribute, default 'polite'.
  *
- * @return {Object} $container The ARIA live region jQuery object.
+ * @return {HTMLDivElement} The ARIA live region jQuery object.
  */
 const addContainer = function( ariaLive ) {
 	ariaLive = ariaLive || 'polite';
@@ -29,7 +29,9 @@ const addContainer = function( ariaLive ) {
 	container.setAttribute( 'aria-relevant', 'additions text' );
 	container.setAttribute( 'aria-atomic', 'true' );
 
-	document.querySelector( 'body' ).appendChild( container );
+	// Disable reason: the querySelector + optional chain may make the expression `null`.
+	// eslint-disable-next-line no-unused-expressions
+	document.querySelector( 'body' )?.appendChild( container );
 	return container;
 };
 

--- a/packages/a11y/src/index.js
+++ b/packages/a11y/src/index.js
@@ -14,14 +14,14 @@ import filterMessage from './filterMessage';
  * Create the live regions.
  */
 export const setup = function() {
-	let containerPolite = document.getElementById( 'a11y-speak-polite' );
-	let containerAssertive = document.getElementById( 'a11y-speak-assertive' );
+	const containerPolite = document.getElementById( 'a11y-speak-polite' );
+	const containerAssertive = document.getElementById( 'a11y-speak-assertive' );
 
 	if ( containerPolite === null ) {
-		containerPolite = addContainer( 'polite' );
+		addContainer( 'polite' );
 	}
 	if ( containerAssertive === null ) {
-		containerAssertive = addContainer( 'assertive' );
+		addContainer( 'assertive' );
 	}
 };
 

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -31,6 +31,7 @@
 		"noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 	},
 	"include": [
+		"./packages/a11y/**/*.js",
 		"./packages/blob/**/*.js",
 		"./packages/dom-ready/**/*.js",
 		"./packages/i18n/**/*.js",

